status.go

```diff
@@ -92,9 +92,15 @@ func StatusFormat() []byte {
 		for i, status := range devs {
 			info, _ := status.desc.GetUsbDeviceInfo()
 
-			fmt.Fprintf(buf, " %3d. %s  %4.4x:%.4x  %-5d %q\n",
+			fmt.Fprintf(buf, " %3d. %s  %4.4x:%.4x  %-5s %q\n",
 				i+1, status.desc.UsbAddr,
-				info.Vendor, info.Product, status.HTTPPort,
```

Comment:
Wow! Why not:

```
port := "-"
if status.HTTPPort != 0 {
    port = strconv.Itoa(status.HTTPPort)
}
```

Reply:
That'll work, too, I guess...
(I am new to go, I am a c++ dev, I thought go is all about anonymous functions and such)

Reply:
No, Go is just a C 2.0, with all interesting C artefacts. For example, slices of array (or subslices of some slice) may share the memory with its parent (or may not, is some of them was reallocated by the **append()** function), so if you modify a subslice, you can modify its parent as well. This is very intuitive for people that come from C, like me, but I don't know how the Python people live with this :-)

So lets don't overcomplicate. Anonymous functions are funny and useful, but not at this case.

This RP now became a mix of port number formatting tweak, that I can easily accept when it will be finished, and version number printing, which is too coupled with git. Lets split these two patches so we can manage them separately.